nixos/modules/config/system-path.nix

```diff
@@ -27,9 +27,7 @@ let
       pkgs.gnutar
       pkgs.gzip
       pkgs.xz
-      pkgs.less
       pkgs.libcap
-      pkgs.nano
```

Comment:
To anyone else reviewing: This is fine since `<nixpkgs/nixos/modules/config/defaults.nix>` sets those as defaults for the new configuration options.